www/front_src/src/Resources/Details/tabs/Details/DetailsCard/GroupChip.tsx

```diff
@@ -0,0 +1,120 @@
+import FilterListIcon from '@mui/icons-material/FilterList';
+import makeStyles from '@mui/styles/makeStyles';
+import IconButton from '@mui/material/IconButton';
+
+import { labelConfigure, labelFilter } from '../../../../translatedLabels';
+import { setCriteriaAndNewFilterDerivedAtom } from '../../../../Filter/filterAtoms';
+import { Group } from '../../../models';
+
+const useStyles = makeStyles((theme) => ({
+  chip: {
+    alignSelf: 'center',
+    display: 'flex',
+  },
+  chipAction: {
+    gridArea: '1/1',
+    maxWidth: theme.spacing(14),
+    minWidth: theme.spacing(8),
+    overflow: 'hidden',
+  },
+  chipHover: {
+    backgroundColor: theme.palette.primary.main,
+    display: 'flex',
+    gap: theme.spacing(0.25),
+    gridArea: '1/1',
+  },
+  chipLabel: {
+    display: 'grid',
+    justifyItems: 'center',
+    minWidth: theme.spacing(7),
+    overflow: 'hidden',
+  },
+}));
+
+interface GroupsChipProps {
+  group: Group;
+  type: string;
+}
+
+const GroupChip = ({ group, type }: GroupsChipProps): JSX.Element => {
+  const theme = useTheme();
+  const classes = useStyles();
+  const { t } = useTranslation();
+
+  const [isHovered, setIsHovered] = React.useState<boolean>(false);
+
+  const setCriteriaAndNewFilter = useUpdateAtom(
+    setCriteriaAndNewFilterDerivedAtom,
+  );
+
+  const filterByGroup = (): void => {
+    setCriteriaAndNewFilter({
+      name: type,
+      value: [group],
+    });
+  };
+  const onMouseEnter = (): void => {
+    setIsHovered(true);
+  };
+
+  const onMouseLeave = (): void => {
+    setIsHovered(false);
+  };
+
+  return (
+    <Grid item className={classes.chip} key={group.id}>
+      <Chip
+        aria-label={`${group.name} Chip`}
+        color="primary"
+        label={
+          <div className={classes.chipLabel}>
+            <Tooltip title={group.name}>
+              <Typography
+                className={classes.chipAction}
+                style={{ color: isHovered ? 'transparent' : 'unset' }}
+                variant="body2"
+              >
+                {group.name}
+              </Typography>
+            </Tooltip>
+            {isHovered && (
+              <Grid className={classes.chipHover}>
+                <IconButton
+                  aria-label={`${group.name} Filter`}
+                  size="small"
+                  style={{ color: theme.palette.common.white }}
+                  title={t(labelFilter)}
+                  onClick={(): void => filterByGroup()}
+                >
+                  <FilterListIcon fontSize="small" />
+                </IconButton>
+                <IconButton
+                  aria-label={`${group.name} Configure`}
+                  size="small"
+                  style={{ color: theme.palette.common.white }}
+                  title={t(labelConfigure)}
+                  onClick={(): void => {
+                    window.location.href = group.configuration_uri as string;
+                  }}
```

Comment:
Extract it to a specific function

Reply:
Ok , i'll add this function
const configureGroup = (): void => {
                    window.location.href = group.configuration_uri as string;
                  };

Reply:
Maybe goToConfigurationPage